text/0021-versioning-releases.md

```diff
@@ -0,0 +1,45 @@
+# Releasing and Versioning of Openmrs SingleSpa Apps
+- Start Date: 2020/01/21
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/1
+
+## Definition
+OpenMRS: refers to the openmrs community.
+
+Micro-Frontends Squad: refers to the OpenMRS Community members responsible for creating and maintaining OpenMRS SPAs, under the OpenMRS Micro-Frontends Project.
+
+OpenMRS SPA: refers to an OpenMRS Micro-Frontend app. An example is openmrs-esm-patient-chart. See https://github.com/openmrs/openmrs-esm-patient-chart
+
+CDN: Content Distribution Network such as jsdelivr. See https://www.jsdelivr.com/
+
+MAJOR Version: version when you make incompatible API changes
+
+MINOR Version: version when you add functionality in a backwards compatible manner
+
+PATCH Version: version when you make backwards compatible bug fixes
+
+## Decision
+OpenMRS SPA applications should have releases that are versioned so that each implementor/distribution can decide when to upgrade to a certain version, containing features that the implementor/distribution needs. 
```

Comment:
I'd delete this line -- SPA Applications already are versioned according to SemVar.